linux_os/guide/services/ssh/ssh_server/sshd_set_max_auth_tries/tests/correct_value.pass.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+SSHD_CONFIG="/etc/ssh/sshd_config"
+
+if grep -q "^MaxAuthTries" $SSHD_CONFIG; then
+	sed -i "s/^MaxAuthTries.*/MaxAuthTries 4/" $SSHD_CONFIG
```

Comment:
Since this is for ``correct_value``, should the ``sshd_max_auth_tries_value`` variable be used?

Reply:
Variables are not supported in the test suite. This test scenario uses the implicit `(all)` profile since there is no metadata `profile`, thus using default values (four in this case).